core/src/stories/foundations/css-reset.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Foundations/CSS Reset" />
+
+# CSS Reset
+
+While our web components look and work as intended out of the box, basic HTML elements like `html`, `p`, `h1`, etc, still need some CSS tweaks to work predictably across browsers.
+
+For this reason we recommend a minimal CSS reset, the included comments explain what each part is for.
+
+The CSS Reset is based on [Joshua Comeaus reset](https://www.joshwcomeau.com/css/custom-css-reset). 
+For an in-depth explanation of each part we recommend reading [this article](https://www.joshwcomeau.com/css/custom-css-reset) on his website.
```

Comment:
Is it necessary to link to same article twice?

Reply:
Not sure, but I think I made it work now without doing that